regulation-worker/internal/delete/api/api.go

```diff
@@ -41,9 +41,8 @@ func (*APIManager) GetSupportedDestinations() []string {
 	return supportedDestinations
 }
 
-// prepares payload based on (job,destDetail) & make an API call to transformer.
-// gets (status, failure_reason) which is converted to appropriate model.Error & returned to caller.
-func (api *APIManager) Delete(ctx context.Context, job model.Job, destination model.Destination) model.JobStatus {
+func (api *APIManager) delete(ctx context.Context, job model.Job, destination model.Destination, attempts int) model.JobStatus {
```

Comment:
As we track the no of retry attempts, `retryAttempts` is better than `attempts`.

Reply:
Done